src/utils/getUserById.ts

```diff
@@ -0,0 +1,5 @@
+import usersFromServer from '../api/users';
+
+export const getUserById = (id: number) => {
+  return usersFromServer.find(user => user.id === id);
```

Comment:
[CODE STYLE] - Avoid using one-letter variable naming. While `user => user.id === id` is not incorrect, it's better to use more descriptive variable names for readability. Consider renaming `user` to `currentUser` or another descriptive name to improve code clarity. Example: `currentUser => currentUser.id === id`.